contracts/BlankArt.sol

```diff
@@ -55,14 +62,23 @@ contract BlankArt is ERC721, EIP712, ERC721URIStorage {
     // pending withdrawals by account address
     mapping(bytes32 => bool) private voucherClaimed;
 
+    // EIP2981
+    struct RoyaltyInfo {
+        address recipient;
+        uint16 bps;
```

Comment:
small point but in the reference implementation here I used prior (https://github.com/dievardump/EIP2981-implementation/blob/main/contracts/ERC2981Base.sol) the struct is 
```struct RoyaltyInfo {
        address recipient;
        uint24 amount;
    }```
I don't know how important it is that these fields match exactly -- but might be worth noting they used a uint24 instead of uint16 for the amount.

Reply:
Thanks for your detailed review! 
I ended up pulling that from one of the manifoldxyz contracts here: https://github.com/manifoldxyz/royalty-registry-solidity/blob/55b1634c0eb63eb1ffd7f6ee36cf09d48731eda4/contracts/overrides/IRoyaltyOverride.sol but am definitely good to match the one you have experience with! uint24 will still allow the struct to fit in 1 slot.

Reply:
Just pushed a commit moving from uint16->uint24. Thanks!